### PR TITLE
Assign roles to level buttons

### LINKED ISSUE #166: National level: in the bottom left box "Surveying" need to be changed
to "Field Surveys"

### LINKED ISSUE #168: State level: Borderlines of the state dissapear
after zooming in further into the state.

### LINKED ISSUE #170: Village level: zoom back out to selected state by clicking on state level

### LINKED ISSUE #173: State level: State level button shall be clickable
State level button shall be clickable and zoom to a random state

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Fix #166 
Fix #170 
Fix #173 
Fix #168 

@catcad, @yelsre - when the user clicks on village button it would perform the following action : 
- coming from national level: select a random state which has OG clusters, then select one of the largest clusters to show
- coming from state level: select one of the largest clusters to show

Are you ok with this logic?

### CODE REVIEW

File: app/static/js/sidebar.js [hunk 1]
```diff
@@ -381,12 +397,36 @@ function adapt_view_to_state_level(previous_level) {
...
+      // Update the states menu list
+      document.getElementById("stateSelect").value = selectedState;
+  };
+  if ((previous_level == "national" || previous_level == "state") && trigger == "button"){
+    //TODO: pick a random cluster among the large ones and display it
+     $.post({
+        url: "/filter-cluster",
+        dataType: "json",
+        data: {"state_name": selectedState},
+        success: function(data){console.log(data);},
```
Comment (Bachibouzouk) [inline app/static/js/sidebar.js:427]: `data` contains the info about the cluster from the view, with that how could I fly to the village @christian-rli ? You can take over the rest of this PR (basically replacing `console.log(data)` by the code which fly to the village).

Comment (christian-rli) [reply app/static/js/sidebar.js:427]: OK. I wanted to start the website, but after installing  flask_wtf I got the following error:

File "../index.py", line 1, in <module>
    from app import create_app
  File "/home/christianh/lokal/240_NESP2/NESP2/app/__init__.py", line 7, in <module>
    from .database import get_random_village
ModuleNotFoundError: No module named 'app.database'

Comment (Bachibouzouk) [reply app/static/js/sidebar.js:427]: Oups I need to add his file

Comment (Bachibouzouk) [reply app/static/js/sidebar.js:427]: Meanwhile do pip install -r requirements.txt

**Pull request merged**